src/city/components/CitiesTable.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+import {
+    Paper,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow
+} from "@mui/material";
+
+export const CitiesTable = () => {
+
+    return (
+        <>
+            <TableContainer component={Paper}>
```

Comment:
It's better to create Table component and then use it like <Table ...props />, right now you have a lot of copy-paste

Reply:
It's better to create Table component and then use it like <Table ...props />, right now you have a lot of copy-paste